### ISSUE TITLE
Only upload resources with embedded claims

### ISSUE THREAD

**Issue by brooksmtownsend** [wasmCloud/wasmcloud-otp#38]
Related to #3, supporting OCI artifacts should also only allow resources with embedded claims

Currently, we allow consumers of the web UI to upload signed wasm files to launch as actors (good) and executables to run as a child process as providers (bad). There are numerous security implications with running an unverified executable, and we should mitigate the majority of those by only allowing providers in a [provider-archive](https://github.com/wasmCloud/wasmCloud/tree/main/crates/provider-archive) or similar format so we can validate claims before starting the executable.

**Comment by brooksmtownsend** [comment]
Closing in favor of https://github.com/wasmCloud/wasmcloud-otp/issues/149
